steering-council-elections.md

```diff
@@ -60,6 +60,10 @@ If a Steering Council member cannot be reached and there is an important time-se
 
 The newly elected member(s) will now act as the Steering Council member in question until the original Steering Council member(s) returns or their term expires.
 
+## Term Limits
+
+Steering Council members cannot serve for more than two consecutive terms.
```

Comment:
I would explicitly clarify that this limit resets if an SC member does not serve for one term.

Reply:
Is that not what consecutive implies?